sw/device/tests/rstmgr_cpu_info_test.c

```diff
@@ -180,18 +182,19 @@ bool test_main(void) {
   dif_rv_core_ibex_crash_dump_info_t dump;
 
   dif_aon_timer_t aon_timer;
+  dt_aon_timer_t aon_timer_dt = kDtAonTimerAon;
```

Comment:
You can just reference this directly in `*_init_from_dt` if you're not doing the casting.